src/asset-platform.js

```diff
@@ -1,11 +1,10 @@
 const { PLATFORM_ARCH } = require("./constants");
 
 const assetPlatform = (fileName) => {
-  if (/.*(mac|darwin|osx).*(-arm).*\.zip/i.test(fileName)) {
-    return PLATFORM_ARCH.DARWIN_ARM64;
-  }
+  if (/.*(mac|darwin|osx).*\.zip$/i.test(fileName)) {
```

Comment:
@MarshallOfSound - thoughts on whether the `-` prefix should also be added here?

ie: `/.*-(mac|darwin|osx).*\.zip$/i`

Reply:
After reviewing the asset and update unit tests I went ahead and updated the regex to prevent incorrectly matching